src/bci_build/package/__init__.py

```diff
@@ -1393,6 +1334,14 @@ def image_type(self) -> ImageType:
     def version_label(self) -> str:
         return str(self.version)
 
+    @property
+    def url(self) -> str:
+        """The default url that is put into the ``org.opencontainers.image.url`` label"""
+        if not self.os_version.is_tumbleweed:
+            return "https://www.suse.com/products/base-container-images/"
+
+        return super().url
```

Comment:
Why do we need this override? We already return the BCI url in line 666 for non-LTSS, non TW containers.

Reply:
already explained earlier. the unit test suite is testing DevelopmentContainers for LTSS, which does not happen in practice, but if it happens then this change is needed. 

alternatively review #1399 please